### ISSUE TITLE
Problème de navigation appli web 

### ISSUE THREAD

**Issue by wyssmarie** [l3miage-jandotdf/Miahoot#1]
Bonjour, 
Nous avons déployé notre application Miahoot sur internet via Firebase. Voici le lien : https://miahoot-jandot.web.app/
Cependant, je n'arrive pas a faire en sorte que dès la connexion on soit redirigé vers la page d'accueil. Pourtant cela fonctionne en local. De plus j'ai l'impression que l'appli n'exécute pas la fonction login quand je me connecte. Pouvez-vous m'aider ?

**Comment by AlexDmr** [comment]
Affichez toujours la conseole de dev du navigateur pour savoir ce qui se passe...
Comme le code est compilé je ne vois pas exactement d'où vient le problème, pouvez vous me renvoyer le lien vers votre Firebase ?

**Comment by wyssmarie** [comment]
https://console.firebase.google.com/project/miahoot-jandot/overview

**Comment by AlexDmr** [comment]
Sans rapport peut être avec ce problème particulier mais vous avez laissé des URL localhost:8080 dans votre code, il faut les remplacer par l'adresse IP de votre serveur Spring.

**Comment by AlexDmr** [comment]
je n'ai pas d'authorisation d'accès à votre projet Firebase.

**Comment by wyssmarie** [comment]
Normalement Floriane vous as ajouté

**Comment by AlexDmr** [comment]
alxdmr2@gmail.com

**Comment by wyssmarie** [comment]
Normalement ça devrait être bon maintenant

**Comment by AlexDmr** [comment]
=> Vous avez réglé le problème des liens vers localhost:8080 dans votre code ?

**Comment by AlexDmr** [comment]
toujours pas accès

**Comment by wyssmarie** [comment]
Non, nous n'avons toujours pas réglé le problème de localhost:8080. Normalement vous devriez recevoir une invitation par mail pour rejoindre le projet firebase. Vous ne l'avez pas ?

**Comment by AlexDmr** [comment]
si c'est bon je suis dessus

**Comment by AlexDmr** [comment]
Difficile à dire d'où vient le problème, ça log bien mais ça lève une erreru dans la console (peut être un problème d'accès aux données, ou du routeur ? ), vous utilisez bien les observables pour gérer tout ça ?

**Comment by wyssmarie** [comment]
Il me semble pas qu'on utilise pour le router mais pour l'acces au données de l'utilisateur oui

**Comment by AlexDmr** [comment]
lien vers le github client ?

**Comment by wyssmarie** [comment]
Si vous parlez du github où il y a le front c'est celui-ci. Sinon le github du back : https://github.com/l3miage-koneze/l3-miage-Miahoot-Groupe3
